projects/addon-doc/components/api/api-item.component.ts

```diff
@@ -81,6 +87,21 @@ export class TuiDocAPIItem<T> implements OnInit {
         this.setQueryParam(value);
     }
 
+    public emitEvent(event: unknown): void {
+        // For more convenient debugging
+        console.info(this.name, event);
```

Comment:
What do you think?
`console.info('DEBUG:', this.name, event);`

Reply:
I think it's better to remove the output to the console altogether, so as not to confuse users :)
`DEBUG:` is more informative, but I'm not sure if it's needed at all, but if it has practical use, then we can leave it, but I have not encountered it yet

Reply:
`I think it's better to remove the output to the console altogether, so as not to confuse users` sure, go ahead

Reply:
> go ahead

Okay, let's delete it completely :)